nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/WebClient.java

```diff
@@ -134,7 +139,11 @@ public B tls(Supplier<Tls> tls) {
          * @return updated builder
          */
         public B channelOptions(SocketOptions channelOptions) {
-            this.channelOptions = channelOptions;
```

Comment:
I don't think we want to modify this method.
If an instance of `SocketOptions` is provided, we use it as-is and that means we discard the `SocketOptions.Builder`.

Reply:
Does it mean that I need to add to JavaDocs to the `readTimeout` and `connectTimeout` shortcuts the notes that they have less priority than `readTimeout` and `connectTimeout` from `channelOptions` and they will be discarded if `channelOptions` will be used during the build of the `WebClient`?